app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
Hmm, I guess I see removing the border as another "processing" step before OCR so I believe it should be in OcrRunnable. It's only 4 arithmetic operations, but we also keep that off the UI Thread if it's in OcrRunnable.

I'd agree to calculating it at CaptureWindow construction, then perhaps also just setting it as a private member variable in OcrRunnable during it's construction.